src/applications/disability-benefits/526EZ/helpers.jsx

```diff
@@ -102,6 +102,43 @@ export const getReservesGuardData = formData => {
   };
 };
 
+/**
+ * Converts the treatment date range into an array of objects from just an object
+ * @param {object} treatmentDateRange object containing from/to date range
+ * @returns {array} of treatmentDateRange's
+ */
+const transformDateRange = treatmentDateRange => [treatmentDateRange];
```

Comment:
What's the reason that this transformation is needed? Isn't each provider facility limited to exactly only one date range?

Reply:
They can have multiple date ranges per provider facility. It's easier for the backend to handle it as an array every time, regardless if there is only 1. Looks something along the lines of this:
```js
treatmentDateRange = [ { from: 'fromDate', to: 'toDate' }, { from: 'fromDate2', to: 'toDate2' } ... ]
```

Reply:
:thinking: Is it a business requirement to allow multiple date ranges per facility? I'm not sure how that would make sense to the veteran.

At any rate, it doesn't look like they can currently enter multiple date ranges. 
![image](https://user-images.githubusercontent.com/12970166/47681470-c743a580-db86-11e8-8fa2-4559f72cbfb3.png)

Reply:
So I probably wasn't very clear here, and your concern with the array of a single dateRange object is valid. The main reasoning behind this is because the API treats it as an array of potential multiple dateRanges. The UI currently does not handle it as such, and as part of MVP it was decided(by our business analysts @Jennywalter ) to only allow them to add one treatmentDateRange as of now, this could change in the future, and the API is setup to support it.

Reply:
![image](https://user-images.githubusercontent.com/22040793/47726323-a46dcb80-dc30-11e8-97ac-64cce5450343.png)
See 4142 ^^

Reply:
I guess my main concern is this is unnecessarily complicated if we decide to stick with the single date range. If it's on the radar to clean this up if necessary after the decision is made, that's good by me.

Reply:
@cvalarida At this point we can't redesign the API to account for taking in a single object, just for the sake of not wrapping it in an array. We're aware that you currently cannot enter in more than one treatmentDateRange, but as this is subject to change, it makes sense to not have to redo *both* the front end and the backend once we need this change implemented.